job/boltjobdb_test.go

```diff
@@ -8,11 +8,15 @@ import (
 )
 
 func TestSaveAndGetJob(t *testing.T) {
+	db := GetBoltDB(testDbPath)
+	cache := NewMemoryJobCache(db, time.Second*60)
+	defer db.Close()
+
 	genericMockJob := getMockJobWithGenericSchedule()
-	genericMockJob.Init()
-	genericMockJob.Save()
+	genericMockJob.Init(cache)
```

Comment:
@ajvb I know the Init method is defined elsewhere, but the constructor method should "Init" the job. My philosophy is that when I get an object it's "good to go".